test/installer/scripts/omsadmin_systest.rb

```diff
@@ -194,11 +202,36 @@ def set_omsagent_proc_limit(val = nil, should_succeed = true)
     return output
   end
 
+  def show_omsagent_proc_limit
+    lstderrprefix='/tmp/show_omsagent_proc_limit'
```

Comment:
Since all of your logging for show_omsagent_proc_limit are log_* methods, they will print out "error" or "warning" or "info" with the message that you can check for. This stdout/stderr split creates more problems than I believe it solves, especially because it creates files in /tmp/ that may conflict when a machine has multiple tests running.

Reply:
Good point.  It's a product of my back and forth learning I fear.